book-my-show/backend/java/bms-monolith/src/main/java/org/lbcc/bms/bms_monolith/common/entity/BaseAuditingEntity.java

```diff
@@ -17,14 +26,27 @@
 @NoArgsConstructor
 @Setter
 @SuperBuilder
+@EntityListeners(AuditingEntityListener.class)
 public abstract class BaseAuditingEntity {
 
     @Id
+    @GeneratedValue(strategy = GenerationType.UUID)
+    @Column(columnDefinition = "CHAR(36)")
     private UUID id;
 
+    @CreatedDate
+    @Column(updatable = false)
     private Instant createdDate;
+
+    @LastModifiedDate
+    @Column(insertable = false)
     private Instant lastModifiedDate;
 
+    @CreatedBy
+    @Column(updatable = false)
     private String createdBy;
+
+    @LastModifiedBy
+    @Column(insertable = false)
     private String lastModifiedBy;
```

Comment:
Add method

@AssertTrue(message = "LastModifiedDate should be after createdDate")
    private boolean isLastModifiedDateAfterCreatedDate() {
        return createdDate == null || lastModifiedDate== null || lastModifiedDate.isAfter(createdDate);
    }